guides/release/components-update/component-arguments-and-html-attributes.md

```diff
@@ -0,0 +1,154 @@
+If you want to _reuse_ a component in multiple places, you'll need a way to
+template out parts of it.
+
+Let's start with two similar but not identical avatar components, that represent
+different users:
+
+```handlebars {data-filename="app/components/received-message/avatar.hbs"}
+<aside>
+  <div class="avatar" title="Tomster's avatar">T</div>
+</aside>
+```
+
+```handlebars {data-filename="app/components/sent-message/avatar.hbs"}
+<aside class="current-user">
+  <div class="avatar" title="Zoey's avatar">Z</div>
+</aside>
+```
+
+The _structure_ of these components is identical, but they have somewhat
+different content (the user's initial) and attributes (the `title` and `class`
+attributes).
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-message">
+        You may have noticed that the <code>is-active</code> class on the
+        received message avatar from the previous chapters is missing here.
+        We'll return to this in the next chapter on
+        <a href="../conditional-content">Conditional Content</a>.
+      </div>
+    </div>
+  </div>
+</div>
+
+
+## Arguments
+
+We can create a component that can be used in both situation by _templating_
+the parts of the HTML that are different.
+
+```handlebars {data-filename="app/components/avatar.hbs"}
+<aside>
+  <div class="avatar" title="{{@title}}">{{@initial}}</div>
+</aside>
+```
+
+The syntax `{{@initial}}` means that the contents inside the `<div>` tag are
+_dynamic_ and will be specified by the `<Avatar>` tag. Likewise, the
+`{{@title}}` syntax means that the contents of the `title` attribute are dynamic
+and will be specified in the same way. We can now replace the received message
+avatar template with a usage of the `<Avatar>` tag providing arguments.
+
+```handlebars {data-filename="app/components/received-message/avatar.hbs"}
+<Avatar @title="Tomster's avatar" @initial="T" />
+```
+
+This code includes the `<Avatar>` component, which expects two _arguments_:
+`@title` and `@initial`.
+
+You are probably familiar with HTML attributes, which tell the _browser_ how to
+draw an HTML element. The syntax `@title=` is similar, but instead of telling
+the _browser_ what to do, it's telling your custom tag what to do.
+
+<div class="cta">
+  <div class="cta-note">
+    <div class="cta-note-body">
+      <div class="cta-note-heading">Zoey says...</div>
+      <div class="cta-note-message">
+        <p>
+          You might be wondering why Ember doesn't use attribute syntax for the
+          component syntax. We'll learn the next section that you can put normal
```

Comment:
Ah!

Imagine that you're coming from an HTML background and we've taught you that Ember components are sort of like custom HTML tags, but you use capital letters for tag names.

You would naturally assume something like:

```hbs
<Avatar title="Tomster's avatar" initial="T" />
```

But we're telling you to use `@title`. It would be natural to wonder why. Do you have any ideas for wording that would make this clearer?

Reply:
Ok, that makes sense. Maybe we should just be explicit here?

> You might be wondering why Ember doesn't use attribute syntax for the component syntax. In other words, why can't we pass in `title=` here, and why do we have to pass in `@title=` instead? In the next section, we'll learn...